### ISSUE TITLE
[Bug] App just not opening

### ISSUE THREAD

**Issue by CloveTwilight3** [Vencord/Vesktop#1025]
### Discord Account

mazeymoos0022

### Operating System

Windows 11

### Package Type

Setup EXE (Installer)

### What happens when the bug or crash occurs?

Title explains it all, I click the app, not opening. Just get this white box. Using version 1.5.4 (latest). Tried reinstalling and also restarting my machine... Still no luck. Been working fine on v1.5.3
![image](https://github.com/user-attachments/assets/bf459a62-74bc-4b52-85f7-1986b3632e67)

### What is the expected behaviour?

Vesktop/Discord should open

### How do you recreate this bug or crash?

1) Open the app... That's all I did... 

### Debug Logs

```
Replace this text with your crash-log. Do not remove the backticks
```

### Request Agreement

- [X] I have searched the existing issues and found no similar issue
- [X] I am using the latest Vesktop and Vencord versions
- [X] This issue occurs on an official release (not just the AUR or Nix packages)

**Comment by x8c8r** [comment]
Seconded

**Comment by Vendicated** [comment]
please provide logs

**Comment by x8c8r** [comment]
> please provide logs

are you gonna tell us where to find them? i cant find any info on where they are stored and for obvious reasons i cant check the console.

**Comment by kamild1996** [comment]
I just had this issue after restarting my PC from a crash. The only logs I could find were in `%appdata%/vesktop/sessionData/Session Storage`. The 000003.log file contained this:
```*®Ë#            version1 
namespace-©&f            
```
And the LOG file contained this:
```
2024/12/31-11:28:57.897 2134 Reusing MANIFEST C:\Users\Kamil\AppData\Roaming\vesktop\sessionData\Session Storage/MANIFEST-000001
```
That's it. I renamed the "vesktop" folder to "_vesktop", relaunched Vesktop, reconfigured it and it works fine now. But it would be nice to know if there's a better way to deal with this.

**Comment by mazeymoos0022** [comment]
> please provide logs

Just adding my file from `C:\Users\MIKI\AppData\Roaming\vesktop\sessionData\Session Storage` (Windows 11)
[000004.log](https://github.com/user-attachments/files/18290358/000004.log)

For other logs, can you please verify the location of where these are stored? I can find them, I just need to know where they are saved to. Also to add I have Vencord on the canary branch of Discord if this is somehow interfearing. (extra info, might not be needed but may be useful, idk)

**Comment by Vendicated** [comment]
you get logs by running vesktop from the command like

![Screenshot_2025-01-02-15-52-40-32_4d38fce200f96aeac5e860e739312e76](https://github.com/user-attachments/assets/dd5becd6-7ec7-4360-847b-e3bb8401a8e2)

**Comment by x8c8r** [comment]
> you get logs by running vesktop from the command like
> 
> ![Screenshot_2025-01-02-15-52-40-32_4d38fce200f96aeac5e860e739312e76](https://private-user-images.githubusercontent.com/45497981/399730717-dd5becd6-7ec7-4360-847b-e3bb8401a8e2.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYwNjU4MjQsIm5iZiI6MTczNjA2NTUyNCwicGF0aCI6Ii80NTQ5Nzk4MS8zOTk3MzA3MTctZGQ1YmVjZDYtN2VjNy00MzYwLTg0N2ItZTNiYjg0MDFhOGUyLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA1VDA4MjUyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ5ODUyZGM5ODMwMDYzZjRmZGNjNzk0YjQzM2Y1YjM3N2YzMTUwMjJjZDMxZTBlZmZhNTBmNjJiMTVlYmUzMTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.gY99PY2Nt61f3TkcQDe9j3Aegnpo0EjMCLlRwfbR7ZA)

here is my log

```
Failed to parse Vesktop state.json: SyntaxError: Unexpected token '', ""... is not valid JSON
    at JSON.parse (<anonymous>)
    at Vr (VCDMain:1:2605)
    at VCDMain:1:3398
    at VCDMain:1:225
    at VCDMain:1:4559
    at VCDMain:1:225
    at Object.<anonymous> (VCDMain:87:331)
    at Module._compile (node:internal/modules/cjs/loader:1484:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1564:10)
    at Module.load (node:internal/modules/cjs/loader:1295:32)
Checking for update
Update for version 1.5.4 is not available (latest version: 1.5.4, downgrade is disallowed).
checkForUpdatesAndNotify called, downloadPromise is null
```

**Comment by Vendicated** [comment]
delete state.json from your vesktop folder (in `%APPDATA%`, `~/.config` or Application Support)

**Comment by x8c8r** [comment]
> delete state.json from your vesktop folder (in `%APPDATA%`, `~/.config` or Application Support)

the error is gone, but the app is still just a white square. left it open for like 30 min nothing happened. the log just talks about the update, same thing as last time.

**Comment by ViliusP** [comment]
I am using cosmic-os (wayland). I tried either to remove `frame: true`  or to add `transparent: false`, but it doesn't change situation. After playing for about an hour, I managed to make it render.

After running `pnpm start`, I got:

```
APPIMAGE env is not defined, current application is not an AppImage
checkForUpdatesAndNotify called, downloadPromise is null
(node:61852) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/viliusp/.config/vesktop/settings/quickCss.css'
    at async open (node:internal/fs/promises:639:25)
(Use `electron --trace-warnings ...` to show where the warning was created)
(node:61852) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

src/gbm_drv_common.c:131: GBM-DRV error (get_bytes_per_component): Unknown or not supported format: 808530000

{
  discordBranch: 'stable',
  richPresence: 'on',
  importSettings: 'on',
  minimizeToTray: 'on'
}
Failed to import settings: Error: ENOENT: no such file or directory, scandir '/home/viliusp/.config/Vencord/settings'
    at readdirSync (node:fs:1506:26)
    at t.readdirSync (node:electron/js2c/node_init:2:13703)
    at WebContents.<anonymous> (VCDMain:87:4114)
    at WebContents.emit (node:events:518:28) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/home/viliusp/.config/Vencord/settings'
}
[arRPC > ipc] listening at /run/user/1000/discord-ipc-0
[arRPC > websocket] listening on 6463
[arRPC > process] started
```

When I tried running with various combinations of the following:

- Deleting `frame: true`;
- Adding `transparent: false`;
- Running with `--ozone-platform-hint=auto --ozone-platform=x11`;
- Running with `--ozone-platform-hint=auto --ozone-platform=wayland --enable-features=WaylandWindowDecorations`;

After several attempts of various combintations, I managed to make the login screen appear, but I couldn't consistently reproduce the result.

After some difficulties and monkey testing, I managed to consistently reproduce it: 

- Delete the configuration files:  `rm -rf ~/.config/vesktop/ `;
- Run with `pnpm start --ozone-platform-hint=auto --ozone-platform=x11`;
- Wait until it loads (for me, this takes about 30–60 seconds);
- A white window should appear.
- Press and hold <kbd>ctrl</kbd>, (mandatory) wait for few seconds and press <kbd>R</kbd>.
- If last step did nothing, repeat it for some times.
- Every other start, should start vesktop correctly, without this white screen.


To try replicate it again, delete configs again  `rm -rf ~/.config/vesktop/ `

This also somehow managed to fix my flatkpak version of Vesktop. So It is probably linked with settings/configurations or electron itself. Also I think, if the login screen isn't rendered, on the next open Vesktop downloads files again (because it loads for some time).  

TL:DR If you have the same problem, **Press and hold <kbd>ctrl</kbd>, wait for few seconds and press <kbd>R</kbd>**. Repeat till it shows something. Also, before you can delete config folder.

**Comment by x8c8r** [comment]
> Can someone who is experiencing this try to build and run with either `frame: true` removed or add `transparent: false`
> 
> https://github.com/Vencord/Vesktop/blob/6c4ecc0d64a843ec942b4fa645c0bfe617b21257/src/main/firstLaunch.ts#L29-L36

worked for me

**Comment by TuxBalazs** [comment]
Clicking on the reset Vencord option in the tray menu, than selecting canary in the installer fixed the issue for me.
